samples/nrf9160/coap_client/README.rst

```diff
@@ -32,13 +45,21 @@ Testing
 
 After programming the sample and all prerequisites to the board, test it by performing the following steps:
 
-1. Connect the USB cable and power on or reset your nRF9160 DK.
-#. Open a terminal emulator and observe that the kit prints the following information::
+1. Connect your nRF9160 DK to the PC using USB cable and power on or reset your nRF9160 DK.
+#. Open a terminal emulator and observe that the following information is displayed::
 
        The nRF CoAP client sample started
-#. Observe that the kit sends periodic CoAP GET requests after it gets LTE connection.
-   The kit sends requests to the configured server (``CONFIG_COAP_SERVER_HOSTNAME``) for a configured resource (``CONFIG_COAP_RESOURCE``).
-#. Observe that the kit either prints the reponse data received from the server or indicates a time-out.
+#. Observe that the following AT commands are displayed on the terminal emulator:
+	   
+	   * :command:`AT+COPS=3,2` which sets format of :command:`+COPS` read command response as numeric
+	   * :command:`AT+COPS?` which reads the current network selection mode and network operator name
+	   * :command:`AT%XCBAND`  which reads the current E-UTRA band
+	   * :command:`AT+CGDCONT?` which configures connection parameters
+	   * :command:`AT+CGACT?` which reads the PDP connections and states
+	
+#. Observe that the discovered IP address of the public CoAP server is displayed on the terminal emulator.
+#. Observe that the nRF9160 DK sends periodic CoAP GET requests to the configured server for a configured resource after it gets LTE connection.
+#. Observe that the sample either displays the response data received from the server or indicates a time-out on the terminal.
 
 Dependencies
 ************
```

Comment:
Should we add link to AT commands ref guide ? Should we add links to infocenter (https://infocenter.nordicsemi.com/index.jsp?topic=%2Fcom.nordic.infocenter.sdk5.v15.3.0%2Flib_iot_coap_addon_observe.html, https://infocenter.nordicsemi.com/topic/sdk_nrf5_v16.0.0/iot_sdk_user_guides_java_coaps.html) Are these links relevant here?

Reply:
We should not link to the nRF5 SDK here. As for the AT commands, I don't think these are relevant for this sample, but providing a link should be no harm.

Reply:
Added a link to AT commands.